### ISSUE TITLE
Rini Individual Ticket - FRQ Showing 2 (Week 8) 

### ISSUE THREAD

**Issue by rkwreck** [rkwreck/team_flappy_bird#33]
My repl (all FRQS are here). This is my menu: https://replit.com/@Rini/FRQWork#Main.java

Directions/Guide to use Repl w/ tangibles: [Link](https://github.com/rkwreck/team_flappy_bird/wiki/Guide-&-Tangibles:-Rini-FRQ-Showing-2-(Week-8))

My learnings/talking points from each FRQ: [Link](https://github.com/rkwreck/team_flappy_bird/wiki/Rini:-FRQ-Learnings-aka-FRQ-Showing)

Github Source: [Link](https://github.com/rkwreck/Repl_FRQs/tree/main)

These are the things I'm required to have. You can check them off as you find them: 

**1. Individual Work: _score individual on completion percentage_.**
- [x] +90% for: 9 FRQs are under management. Management includes all the individual work from FRQ Showing 1.
- [x] +10% for: At least 2 FRQS with input and output
- [x] Tangible evidence

**Comment by ridhimainukurti** [comment]
Rini's Grading: 

(+): I thought you did a great job with each FRQ providing input and output for most of them which was nice to see considering you surpassed the requirement of 2. 

(-): I think for next time, it would be nice if there were more of an explanation for FRQ guidance instead of it just being in bullet form. -0.15 points 

(+): Your review ticket contains all the information making it easy for the grader as well as the guidance readme is very easy to follow along especially with the pictures added. 

Total Grade: 8.85/9

**Comment by rkwreck** [comment]
Comparison of 3 FRQS:

Unit 2 - My crossover partner has input for every sequence, whereas I ask input for the last two-thirds of the FRQ. An improvement for me would me to have more inputs for this question.

Unit 3 - My crossover partner created more than just 4 options for food. However, they prompt the user to "Enter a string for option2" without explaining what that means. An improvement for me could be to add more options as well. A plus improvement is that throughout the FRQ or before the FRQ begins, I have print statements that explain what the code will do so the user understands what's going on. what they need to input, and what is the purpose of that input.

Unit 5 - My crossover partner asks inputs for the host age but doesn't use it. In comparison, I use all the information I asked inputs for. 

My Reflection:
One of my strengths is that all of my FRQs work. Both my Repl and Review Ticket were organized as well. The guide, Repl, and Review Ticket were easy to follow. I met and exceeded all requirements. One weakness was that I did not include tangibles in the FRQ Learnings/Highlighted talking points. This was something my partner included that I think was a nice addition. Also, my READMe for my Github source could have had more explanation, like my partner's. I made this change.
